### ISSUE TITLE
相关资源

### ISSUE THREAD

**Issue by ethan-funny** [ethan-funny/explore-machine-learning#1]
# 石晓文的学习日记

- [princewen/tensorflow_practice: tensorflow的一些实战练习](https://github.com/princewen/tensorflow_practice)

---

- [推荐系统遇上深度学习(一)--FM模型理论和实践<br>](https://www.jianshu.com/p/152ae633fb00)
- [推荐系统遇上深度学习(二)--FFM模型理论和实践<br>](https://www.jianshu.com/p/781cde3d5f3d)
- [推荐系统遇上深度学习(三)--DeepFM模型理论和实践<br>](https://www.jianshu.com/p/6f1c2643d31b)
- [推荐系统遇上深度学习(四)--多值离散特征的embedding解决方案<br>](https://www.jianshu.com/p/4a7525c018b2)
- [推荐系统遇上深度学习(五)--Deep&Cross Network模型理论和实践<br>](https://www.jianshu.com/p/77719fc252fa)
- [推荐系统遇上深度学习(六)--PNN模型理论和实践<br>](https://www.jianshu.com/p/be784ab4abc2)
- [推荐系统遇上深度学习(七)--NFM模型理论和实践 - 简书](https://www.jianshu.com/p/4e65723ee632)
- [推荐系统遇上深度学习(八)--AFM模型理论和实践 - 简书](https://www.jianshu.com/p/83d3b2a1e55d)
- [推荐系统遇上深度学习(九)--评价指标AUC原理及实践 - 简书](https://www.jianshu.com/p/4dde15a56d44)
- [推荐系统遇上深度学习(十)--GBDT+LR融合方案实战 - 简书](https://www.jianshu.com/p/96173f2c2fb4)
- [推荐系统遇上深度学习(十一)--神经协同过滤NCF原理及实战 - 简书](https://www.jianshu.com/p/6173dbde4f53)
- [推荐系统遇上深度学习(十二)--推荐系统中的EE问题及基本Bandit算法 - 简书](https://www.jianshu.com/p/95b2de50ce44)
- [推荐系统遇上深度学习(十三)--linUCB方法浅析及实现 - 简书](https://www.jianshu.com/p/e0e843d78e3c)
- [推荐系统遇上深度学习(十四)--《DRN:A Deep Reinforcement Learn... - 简书](https://www.jianshu.com/p/c0384b213320)
- [推荐系统遇上深度学习(十五)--强化学习在京东推荐中的探索 - 简书](https://www.jianshu.com/p/b9113332e33e)
- [推荐系统遇上深度学习(十六)--详解推荐系统中的常用评测指标 - 简书](https://www.jianshu.com/p/665f9f168eff)

--- 

- [实战深度强化学习DQN-理论和实践](https://www.jianshu.com/p/10930c371cac)
- [DQN三大改进(一)-Double DQN](https://www.jianshu.com/p/fae51b5fe000)
- [DQN三大改进(二)-Prioritised replay](https://www.jianshu.com/p/db14fdc67d2c)
- [DQN三大改进(三)-Dueling Network](https://www.jianshu.com/p/b421c85796a2)
- [深度强化学习-Policy Gradient基本实现](https://www.jianshu.com/p/2ccbab48414b)
- [深度强化学习-Actor-Critic算法原理和实现](https://www.jianshu.com/p/6fe18d0d8822)
- [深度强化学习-DDPG算法原理和实现](https://www.jianshu.com/p/6fe18d0d8822)
- [Pointer-network理论及tensorflow实战](https://www.jianshu.com/p/2ad389e91467)
- [对抗思想与强化学习的碰撞-SeqGAN模型原理和代码解析 - 简书](https://www.jianshu.com/p/de4e913e0580)
- [探秘多智能体强化学习-MADDPG算法原理及简单实现 - 简书](https://www.jianshu.com/p/4e4e35d80137)

---

- [各种排序算法的分析及java&python实现 - 简书](https://www.jianshu.com/p/673d70209dd2)
- [L1正则化及推导 - 简书](https://www.jianshu.com/p/2f60e672d4f0)
- [简单的Seq2Seq实现作对联 - 简书](https://www.jianshu.com/p/83443b2baf27)

# 机器学习荐货情报局

- [计算广告CTR预估系列(一)--DeepFM理论](https://mp.weixin.qq.com/s?__biz=MzU0NDgwNzIwMQ==&mid=2247483673&idx=1&sn=256e57219c8d577c61f25221c346053c&chksm=fb77c157cc004841581591543041044f490825c59360dd485a4d005b0c3823feb0d0c39bf9e3&scene=21#wechat_redirect)
- [计算广告CTR预估系列(二)--DeepFM实践](https://mp.weixin.qq.com/s?__biz=MzU0NDgwNzIwMQ==&mid=2247483677&idx=1&sn=5bf0ac27124f57553cc8c17aa48664c7&chksm=fb77c153cc0048451db16d3ad2a6a6a0c7a001e9277b4a53caa2dd45cc457dac724e9e31a8a8&scene=21#wechat_redirect)
- [计算广告CTR预估系列(三)--FFM理论与实践](https://mp.weixin.qq.com/s?__biz=MzU0NDgwNzIwMQ==&mid=2247483685&idx=1&sn=36de5b8814c7a1ca5d5a19315b3f1ed1&chksm=fb77c16bcc00487d937ca5c10a0682feecd8a1bcd4bc9ddbb13f21ccd3b353439c8ee724ff14&scene=21#wechat_redirect)
- [计算广告CTR预估系列(四)--Wide&Deep理论与实践](https://mp.weixin.qq.com/s?__biz=MzU0NDgwNzIwMQ==&mid=2247483689&idx=1&sn=c6e55677fe4ee1983e8f51fb61dffab5&chksm=fb77c167cc004871347a79fddb1c70d44f2b4bb54cbf09c3d1ecc5473ff8653802354d65bb8d&scene=21#wechat_redirect)
- [计算广告CTR预估系列(五)--阿里Deep Interest Network理论](https://mp.weixin.qq.com/s?__biz=MzU0NDgwNzIwMQ==&mid=2247483704&idx=1&sn=2b80e3def93056e4afb39cc1e744d18a&chksm=fb77c176cc0048607a4f54c787aaf254fc8bd0e852c6c3cb4baa7546073f27e706244391b35e&mpshare=1&scene=1&srcid=0618eac6GkQ01UaGE7urAQol&key=b13b33091d0d698d91e66e990d4a166148b107298f2df8ceaa001774df69cd5405b24b297874d20126666d678747bf36762995489ebb6dfa34a7600e59dce960d5b8756ccca6d3a4d85318892c413fea&ascene=0&uin=OTA5MDI2NzA0&devicetype=iMac+MacBookPro10%2C1+OSX+OSX+10.11.5+build(15F34)&version=12020810&nettype=WIFI&lang=zh_CN&fontScale=100&pass_ticket=A7B8%2Bx22r1oAz%2BppDJkkoAOAkkh9mtcS32uxhMjl%2B1Dc7UufH%2Bq3ZzaYJIe8EbvS)
- [计算广告CTR预估系列(六)--阿里Mixed Logistic Regression](https://mp.weixin.qq.com/s?__biz=MzU0NDgwNzIwMQ==&mid=2247483707&idx=1&sn=5810c525e2880edb795543d5b8bd4aa2&chksm=fb77c175cc0048631a92201e5d4fffcf514f683540cf763a89626f99fc6bba66ac962fa3a07a&scene=21#wechat_redirect)
- [计算广告CTR预估系列(七)--Facebook经典模型LR+GBDT理论与实践](https://mp.weixin.qq.com/s?__biz=MzU0NDgwNzIwMQ==&mid=2247483711&idx=1&sn=14e8d906d84de78b249510b33d423b89&chksm=fb77c171cc0048670169bda56327473f29b60c30187b67812261d13fc6518762f88611c61949&scene=21#wechat_redirect)
- [计算广告CTR预估系列(八)--PNN模型理论与实践](https://mp.weixin.qq.com/s?__biz=MzU0NDgwNzIwMQ==&mid=2247483719&idx=1&sn=ab9b912145c94ef299bc8484372794e9&chksm=fb77c109cc00481f47664515bb4f1ef6fefbc598f88693888572aff5010d2637398a260e1e8a&mpshare=1&scene=1&srcid=0618AnSLKyHKgTHvW79bgfig&key=3523bcf4cfdb43a6e1f276e9342cca9f7fccc3af705c09bafb1ac31550df01a0c825c33171d2ff44a40c09c2006befe377cd7ad3f81ddf02e2486f6f96ec6e2f15ad27fe2df5a498d30daa9853696e7f&ascene=0&uin=OTA5MDI2NzA0&devicetype=iMac+MacBookPro10%2C1+OSX+OSX+10.11.5+build(15F34)&version=12020810&nettype=WIFI&lang=zh_CN&fontScale=100&pass_ticket=A7B8%2Bx22r1oAz%2BppDJkkoAOAkkh9mtcS32uxhMjl%2B1Dc7UufH%2Bq3ZzaYJIe8EbvS)

--- 

- [Batch Normalization，会其意知其形](https://mp.weixin.qq.com/s?__biz=MzU0NDgwNzIwMQ==&mid=2247483668&idx=1&sn=79f6ef661a01cdf137edf235c974e6bc&chksm=fb77c15acc00484ce0deaf230e88e6dd1f377840aa63bcd7bba65e1de78fd40c68926f8cae6c&mpshare=1&scene=1&srcid=0618hNfRdMUBDaZlvTIfzRNq&key=92a2af92b5daeeadd8dcc7ed2cf3a53cff90cce55596223a07bab3396afec68bb96abbf98e303816ede74a00c8c419fc80856b3a018539d3b194a76a7ee740098cbd2699d45f9091293d6dbcad45787a&ascene=0&uin=OTA5MDI2NzA0&devicetype=iMac+MacBookPro10%2C1+OSX+OSX+10.11.5+build(15F34)&version=12020810&nettype=WIFI&lang=zh_CN&fontScale=100&pass_ticket=A7B8%2Bx22r1oAz%2BppDJkkoAOAkkh9mtcS32uxhMjl%2B1Dc7UufH%2Bq3ZzaYJIe8EbvS)
- [机器学习梯度下降优化算法](https://mp.weixin.qq.com/s?__biz=MzU0NDgwNzIwMQ==&mid=2247483698&idx=1&sn=e176138938cf88fca5a8e8104d2a6a6a&chksm=fb77c17ccc00486a2147e559cd198b852754230e29e7069ba690f5c22bc2ef2d26caaa571050&mpshare=1&scene=1&srcid=0618DqilAMxZcTl8WQ7G20LP&key=dd6054e4a096da288903f5e5c4b62b3f8810d9d49a631927bb39f13e820ec673cdb5da16a46d4a0db466fae381a470fe868f7b7466177734aa415460a4b87577a9ba63c1247d0178dde29c185706064f&ascene=0&uin=OTA5MDI2NzA0&devicetype=iMac+MacBookPro10%2C1+OSX+OSX+10.11.5+build(15F34)&version=12020810&nettype=WIFI&lang=zh_CN&fontScale=100&pass_ticket=A7B8%2Bx22r1oAz%2BppDJkkoAOAkkh9mtcS32uxhMjl%2B1Dc7UufH%2Bq3ZzaYJIe8EbvS)
